composer.json

```diff
@@ -11,18 +11,18 @@
   "type": "library",
   "homepage": "https://github.com/apimatic/core-lib-php",
   "require": {
-    "php": ">=7.2 <8.2",
+    "php": ">=7.2 <8.3",
     "ext-json": "*",
     "ext-curl": "*",
     "ext-dom": "*",
     "ext-libxml": "*",
-    "apimatic/core-interfaces": "^0.1.0",
+    "apimatic/core-interfaces": "dev-3-php-8.2-support",
```

Comment:
This change must also be intended for testing, use the correct core-interfaces version that we are going to release